_episodes/02-image-basics.md

```diff
@@ -70,6 +75,16 @@ let's start our exploration with 15 pixels in a 5 X 3 matrix with 2 colours and
 work our way up to that complexity,
 but first the necessary imports.
 
+> ## Matrices, arrays, images and pixels
+> The **matrix** is mathematical concept - numbers evenly arranged in a rectangle. This can be a two dimensional rectangle, 
+> like the shape of the screen you're looking at now. Or it could be a three dimensional equivalent, a cuboid, or have 
+> even more dimensions, but always keeping the evenly spaced arrangement of numbers. In computing, **array** refers 
+> to a structure in the computer's memory where data is stored in evenly-spaced **elements**. This is strongly analogous 
+> to a matrix. A `numpy` array is a **type** of variable (a simpler example of a type is an integer). For our purposes, 
+> the distinction between matrices and arrays is not important, we don't really care how the computer arranges our data 
+> in its memory. The important thing is that the computer stores values describing the pixels in images, as arrays. And 
+> the terms matrix and array can be used intechangably.
+{: .callout}
 ~~~
 """
```

Comment:
```suggestion
work our way up to that complexity.

> ## Matrices, arrays, images and pixels
> The **matrix** is mathematical concept - numbers evenly arranged in a rectangle. This can be a two dimensional rectangle, 
> like the shape of the screen you're looking at now. Or it could be a three dimensional equivalent, a cuboid, or have 
> even more dimensions, but always keeping the evenly spaced arrangement of numbers. In computing, **array** refers 
> to a structure in the computer's memory where data is stored in evenly-spaced **elements**. This is strongly analogous 
> to a matrix. A `numpy` array is a **type** of variable (a simpler example of a type is an integer). For our purposes, 
> the distinction between matrices and arrays is not important, we don't really care how the computer arranges our data 
> in its memory. The important thing is that the computer stores values describing the pixels in images, as arrays. And 
> the terms matrix and array can be used intechangably.
{: .callout}

First, the necessary imports:
~~~
"""
```

Reply:
Thanks @quist00 - a small change and a huge improvement to flow!

Reply:
Although I've messed up the suggestion feature here!